cmd/kusk/cmd/generate.go

```diff
@@ -299,3 +239,56 @@ func init() {
 
 	apiTemplate = template.Must(template.New("api").Parse(templates.APITemplate))
 }
+
+var generateDescription = `Description:
+
+Generate accepts your OpenAPI spec file as input either as a local file or a URL and generates a Kusk 
+Gateway compatible API resource that you can apply directly into your cluster. 
+
+It does this via the x-kusk extension which will be added automatically if one is not already set. It will 
+set the upstream service, namespace and port to the flag values passed, respectively, and set the rest of 
+the settings to defaults.
+
+If the x-kusk extension is already present, it will override the upstream service, namespace and port to 
+the flag values passed, respectively, and leave the rest of the settings as they are.`
+
+var generateHelp = `
+No name specified:
+kusk api generate \
+	-i spec.yaml \
+	--envoyfleet.name kusk-gateway-envoy-fleet \
+	--envoyfleet.namespace kusk-system
+
+In the above example, kusk will use the openapi spec info.title to generate a manifest name and leave the existing
+x-kusk extension settings
+
+No API namespace specified:
+kusk api generate \
+	-i spec.yaml \
+	--name httpbin-api \
+	--upstream.service httpbin \
+	--upstream.port 8080 \
+	--envoyfleet.name kusk-gateway-envoy-fleet
+
+In the above example, as --namespace isn't defined, it will assume the default namespace.
+
+Namespace specified:
```

Comment:
I would remove this example and the "No API namespace specified" one? I think the objective of the examples is that the users can see a couple of scenarios, and they can then relay to the help section